### PR TITLE
LPS-204046 Add selected object folder name to the URL to correctly redirect the user back from Model Builder to the folder listing

### PR DESCRIPTION
**Pull request by danvb112**
Related Issue: https://liferay.atlassian.net/browse/LPS-204046

### CODE REVIEW

**Comment by danvb112** [comment]
ci:test:sf

**Comment by rebsilva** [comment]
just started reviewing :cherry_blossom:

File: modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ModelBuilder/EditObjectFolder.tsx [hunk 1]
```diff
@@ -208,6 +209,14 @@ export default function EditObjectFolder({
 		// eslint-disable-next-line react-hooks/exhaustive-deps
 	}, [selectedObjectFolder.objectFolderItems?.length]);
 
+	Liferay.on('beforeNavigate', () => {
+		updatePreviousURLParam('objectFolderName', objectFolderName);
+	});
+
+	window.addEventListener('popstate', () => {
+		updatePreviousURLParam('objectFolderName', objectFolderName);
+	});
```
Comment (rebsilva) [inline modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ModelBuilder/EditObjectFolder.tsx:218]: hey @danvb112, just wanted to check with you if its okay to this piece of code to be outside a function or a `useEffect` for exemple, won't it be called every single time a change is made on the page? or this was really your intention?

**Review by rebsilva — COMMENTED** [review]
just made a question, if you think it doesn't make sense on this case let me know so i can approve the PR, thanks!

File: modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ModelBuilder/EditObjectFolder.tsx (hunk 1 above)
Comment (rebsilva) [reply modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ModelBuilder/EditObjectFolder.tsx:218]: he explained to me that this won't affect the behavior, so i will approve the PR!

**Review by rebsilva — APPROVED** [review]
LGTM! :tada:

**Comment by rebsilva** [comment]
ci:test:relevant

**Comment by danvb112** [comment]
ci:forward:force

**Pull request closed without merging**